sequencer/node/src/node.rs

```diff
@@ -140,15 +147,26 @@ impl Node {
                         for (i, m) in batch_txs.into_iter().enumerate() {
                             let starknet_tx = rpc::InvokeTransactionV1::from_bytes(&m);
                             info!("Message {i} in {:?} is of tx_type {:?}", p, starknet_tx);
-
                             let n = 10_usize;
```

Comment:
Can we get the `n` from the call data? Should be simple enough right now. It can be added in a subsequent PR as well

Reply:
I'm thinking of doing this in a future refactor, so we can merge this changes and have cairo native in main. What do you think about it?

Reply:
Sounds good